site/content/blogs/release-v127.en.md

```diff
@@ -87,11 +87,76 @@ Continue with the deployment? (y/N)
 
 If the diff looks good to you, enter "y" to deploy. Otherwise, enter "N" to make adjustments as needed!
 
-## Enable multiple listeners and routing rules for Load Balancers
 
-### Add multiple host-based or path-based routing rules to your Application Load Balancers
+## Enable multiple listeners and listener rules for Load Balancers
+You can now configure additional listener rules for Application Load Balancer as well as additional listeners for 
+Network Load Balancer.
+
+### Add multiple host-based or path-based listener rules to your Application Load Balancer
+We can configure additional listener rules for ALB via new field [`http.additional_rules`](../docs/manifest/lb-web-service.en.md#http-additional-rules). 
+It is as easy as configuring your `http` field. Let's learn through an example. 
+
+Say we want to expand the basic manifest such that it opens up port 8081 on the main service container, and 8082 on the sidecar container, in addition to the existing `image.port` 8080.
```

Comment:
Can we modify this sentence as well to reflect the purpose of "handling different traffics" instead of "exposing ports"?